core/testing/runner.odin

```diff
@@ -63,6 +65,21 @@ get_log_level :: #force_inline proc() -> runtime.Logger_Level {
 	}
 }
 
+get_memory_log_level :: #force_inline proc() -> runtime.Logger_Level {
+	when ODIN_DEBUG {
+		// Always use .Debug in `-debug` mode.
+		return .Debug
+	} else {
```

Comment:
Is there a reason for this to default to Debug in `-debug` mode? The reason why general logging forces it to Debug is as a fallback to catch every possible log message, but I'm not sure why memory logging would do this, especially if there's an intent to catch leaks and errors by virtue of setting a higher log level.

Reply:
Not really, no. I just copy/pasted it with minimal edits.